packages/back-end/src/integrations/Athena.ts

```diff
@@ -45,4 +47,11 @@ export default class Athena extends SqlIntegration {
   ensureFloat(col: string): string {
     return `1.0*${col}`;
   }
+
+  async getInformationSchema(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    dataSourceType: DataSourceType
+  ): Promise<InformationSchema[] | null> {
```

Comment:
I should probably make dataSourceType optional in `SQLIntegration` and just add a check to every datasource type that doesn't yet support having an `informationSchema`.